src/view/RaceGameView.js

```diff
@@ -0,0 +1,40 @@
+import RaceGameController from '../controller/RaceGameController.js';
+import { MESSAGE, POSITION_MARKER, WINNER_SEPERATOR } from '../constants/UI.js';
+
+class RaceGameView {
+  #io;
+
+  constructor (io) {
+    this.#io = io;
+  }
+
+  async getCarNames () {
+    const input = await this.#io.in(MESSAGE.PLEASE_INPUT_CAR_NAMES);
+    return RaceGameController.onGetRaceCarNames(input);
+  }
+
+  async getIteration () {
+    const input = await this.#io.in(MESSAGE.PLEASE_INPUT_ITERATION_NUMBER);
+    return RaceGameController.onGetIteration(input);
+  }
+
+  showStatus (cars) {
+    const outputMessages = [];
+    for (const car of cars) {
+      const { name, move } = car;
+      outputMessages.push(`${name} : ${POSITION_MARKER.repeat(move)}\n`);
+    }
+    this.#io.out(outputMessages.join(''));
+  }
```

Comment:
고차함수를 써서 라인을 줄이고
join까지 메세지 포매팅 블록 안에 포함시키면
27번 라인의 가독성이 더 좋아질 것 같아요! 😊

```suggestion
  showStatus(cars) {
    const outputMessages = cars
      .map(({ name, move }) => `${name} : ${POSITION_MARKER.repeat(move)}\n`)
      .join('');
    this.#io.out(outputMessages);
  }
```

Reply:
LGTM ! 감사합니다!